srtcore/common.h

```diff
@@ -614,7 +614,7 @@ class CSeqNo
    /// VALUE for any other purpose. It is not meant to be the
    /// distance between two sequence numbers.
    ///
-   /// Example: to check if (seq1 %> seq2): seqcmp(seq1, seq2) > 0.
+   /// Example: to check if (seq1 > seq2): seqcmp(seq1, seq2) > 0.
```

Comment:
This `%>` is here for a reason. It's an alleged "later than" operator that is explained elsewhere.